theory-test-start-admission/integration-test/AdmissionDAO.integration.js

```diff
@@ -1,35 +0,0 @@
-/*
```

Comment:
While I see you have added the unit test,  I still think the integration test is needed in this case. Otherwise you can not guarantee your DAO is working against the DynamoDB.